### PR TITLE
Adds *pee and *poo

### PR DESCRIPTION
**Pull request by Iamgoofball**
see title

### CODE REVIEW

**Comment by F-OS** [comment]
no. goof. stop.

**Comment by kevinz000** [comment]
Dude, Goofball.
STOP.

**Comment by kevinz000** [comment]
let me rephrase that
fuck off.

**Comment by F-OS** [comment]
Banned from tg github, The PR.

**Comment by SweaterKittens** [comment]
+/datum/emote/living/carbon/human/unspeakable
 +	key = "eat my anus"
 +	key_third_person = "fuck me daddy"
 +	message = "daddy taught us not to be ashamed of our dicks"


What the actual fuck, Goof?

**Comment by WJohn** [comment]
he's legitimately gone insane

**Comment by ExcessiveUseOfCobblestone** [comment]
put me in the screencap

**Comment by kevinz000** [comment]
can i permaban goof for rule 8 if this goes through?

**Comment by F-OS** [comment]
@kevinz000 yes please do.

**Comment by JJRcop** [comment]
Um this gibs you if you try to use either of them with a 1 in a million chance of you getting a pulse destroyer rifle instead.

**Comment by Shadowlight213** [comment]
@kevinz000 Yes

**Comment by Kraseo** [comment]
**FOOOOOOOOOOOOOOOOOOORE!**

**Comment by dannno** [comment]
Hole in one!

**Comment by sawrge** [comment]
i am golf ball

**Comment by Arianya** [comment]
Can we just program a Poly-style bot to open PRs with procedurally generated titles and randomized code changes?

You know, automate goof out of a job.

**Comment by F-OS** [comment]
@Arianya Make sure the bot adds lots of horrible code and make it swear a lot and it might work!

**Comment by Incomptinence** [comment]
DAWNING OF A NEW ERA HE WILL MAKE TG SHITE AGAIN!

**Comment by F-OS** [comment]
HE IS GONNA BUILD A WALL AND MAKE THE MAINTAINERS PAY FOR IT!

**Comment by Arianya** [comment]
@F-OS I'm not sure I can make the bot react believably upset over any minor criticism. The idea might have to be shelved.

**Comment by F-OS** [comment]
@Arianya Yeah good point.

**Comment by Incomptinence** [comment]
@F-OS HE'S GONNA SHIT A WALL!

**Comment by F-OS** [comment]
@tgstation/commit-access I think this pr has lived for long enough.

**Comment by F-OS** [comment]
Thank you.

**Comment by optimumtact** [comment]
lets do it

**Comment by kevinz000** [comment]
Oranges and here I thought you were a sane man.

**Comment by vcordie** [comment]
what the fuck, oranges.

**Comment by F-OS** [comment]
NO

**Comment by PKPenguin321** [comment]
I'm ready

File: code/modules/mob/living/carbon/human/emote.dm [hunk 1]
```diff
@@ -137,6 +137,31 @@
 	update_body()
 
 
+/datum/emote/living/carbon/human/unspeakable
+	key = "eat my anus"
+	key_third_person = "fuck me daddy"
+	message = "daddy taught us not to be ashamed of our dicks"
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/human/emote.dm]: now this is something I can get behind

**Comment by Durkel** [comment]
ABSOLUTE MADMAN

**Comment by PKPenguin321** [comment]
HAHAHAGHAHAGAGAFARASAGAA

**Comment by kevinz000** [comment]
holy fucking shit you scared the living fucking shit out of me mate.

**Comment by F-OS** [comment]
Holy shit i thought tg had just started sliding down the slippery slope, good troll!

**Comment by HankMoodyExperience** [comment]
memes

**Comment by ExcessiveUseOfCobblestone** [comment]
meme

**Comment by gbasood** [comment]
the fall of /tg/ starts here

**Comment by improvedname** [comment]
We are goonstation now

**Comment by peoplearestrange** [comment]
Was this legitimately merged?
Fuck me I left at the right time...

**Comment by RemieRichards** [comment]
@peoplearestrange PAS...

**Comment by peoplearestrange** [comment]
Oh.. right, file changes...

**Comment by ClumsyAlcoholic** [comment]
goofy

**Comment by LetterN** [comment]
// ayy lmao

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -191,3 +166,4 @@
 		dna.species.mutant_bodyparts |= "wings"
 	update_body()
 
+//Ayy lmao
```
Comment (LetterN) [inline code/modules/mob/living/carbon/human/emote.dm]: ayy lmao

**Comment by AlcaroIsAFrick** [comment]
me irl
